frontend/public/components/deploy-image.tsx

```diff
@@ -312,23 +312,24 @@ export class DeployImage extends React.Component<DeployImageProps, DeployImageSt
           </div>
           <p>Deploy an existing image from an {/*image stream tag or */} image registry.</p>
           <div className="form-group co-deploy-image__image-name">
-            <label htmlFor="imageName">Image Name</label>
+            <label htmlFor="image-name">Image Name</label>
             <div className="input-group">
               <input className="form-control"
                 type="search"
                 onChange={this.onImageNameChange}
                 onKeyDown={this.onKeyPress}
                 value={this.state.imageName}
-                id="imageName"
-                name="imageName" />
+                id="image-name"
+                name="image-name"
```

Comment:
`id` should generally be kebab case (`image-name`), but `name` should be camel case (`imageName`)